scripts/globals/items/zanmato_+1.lua

```diff
@@ -0,0 +1,20 @@
+-----------------------------------------
+-- ID: 21966
+-- Item: Zanmato +1
+-- Additional Effect: Dispel (Current dispel rate is guesstimate)
+-----------------------------------------
+require("scripts/globals/msg")
+require("scripts/globals/status")
+-----------------------------------
+
+function onAdditionalEffect(player, target, damage)
+    local chance = 5
+    if math.random(100) <= chance then
+        local effect = target:dispelStatusEffect()
+        if effect ~= dsp.effect.NONE then
+            return dsp.subEffect.DISPEL, dsp.msg.basic.ADD_EFFECT_DISPEL, effect
```

Comment:
AHEM. I keep asking ppl not to add more item scripts for these.
https://github.com/DarkstarProject/darkstar/commit/888be1c596477500fad30ad3e0418d8abd730a9e
please send anything related to additional effects over to that branch instead. (it actually works just needs script cleaned up and sql added)
to add teh add effect all you have to do is set mods there, no scripting needed on the item.

Reply:
PR on hold pending changes to the 2nd pr to add effect branch, agreed on a compromise that lets this one merge as is, then an extra commit fixes the differences on the other branch

Reply:
@TeoTwawki Once you merge to master, I will remove the additional item, zanmato_+1.lua, and add the effect to item_mods as modid 431 with value 6, and mod 501 with value 5. Then it can be merged to the add effects rework matching that style.

Reply:
I was waiting for you to show up on discord tonight for this coz I saw it was teh same branch pr'd to both, merging now. it'll work out the same as what we had planned which was pulling master back into addEffect again